app/utils/bundle-importer.js

```diff
@@ -70,11 +70,34 @@ YUI.add('bundle-importer', function(Y) {
     },
 
     /**
-      Fetch the dry-run output from the Deployer.
+      Fetch the dry-run output from the Guiserver.
 
       @method fetchDryRun
+      @param {String} bundleYAML The bundle file contents.
     */
-    fetchDryRun: function() {},
+    fetchDryRun: function(bundleYAML) {
+      this.env.getChangeSet(bundleYAML, this._handleFetchDryRun.bind(this));
+    },
+
+    /**
+      Handles the dry run response.
+
+      @method _handleFetchDryRun
+      @param {Object} response The processed response data.
+    */
+    _handleFetchDryRun: function(response) {
+      if (response.err) {
```

Comment:
Now the api sends both error messages down the same `.err` object as per the comment above. The notifications are shown only for a brief time and the user needs to be watching to see them. So the extra information available by the console I think is valuable. When we get a better notification system then we can revisit.